docs/TutorialReact.md

```diff
@@ -109,10 +112,21 @@ jest.
       "<rootDir>/node_modules/react-dom",
       "<rootDir>/node_modules/react-addons-test-utils",
       "<rootDir>/node_modules/fbjs"
+    ],
+    "modulePathIgnorePatterns": [
+      "<rootDir>/node_modules/"
     ]
   }
 ```
 
+You'll also need to add a `.babelrc` file:
+
+```
+{
+  "presets": ["es2015", "react"]
```

Comment:
I agree that the mention of the `.babelrc` file along with the presets is something that is currently missing from the documentation.